src/main/resources/avro/variants.avdl

```diff
@@ -94,48 +147,44 @@ record CallSet {
 }
 
 /**
-A `Call` represents the determination of genotype with respect to a
-particular variant. It may include associated information such as quality
-and phasing. For example, a call might assign a probability of 0.32 to
-the occurrence of a SNP named rs1234 in a call set with the name NA12345.
+Represents a `Genotype` integrating over the `Call`s belonging to a particular
+`Variant` in a `CallSet`. Analogous to a `Variant`-level `Call`, as might be
+found in VCF.
+
+Mostly useful for importing VCF annotations, or for giving joint distributions
+on copy numbers of different `Call`s referencing a `Variant` (like genotype
+likelihoods).
 */
-record Call {
+record Genotype {
 
   /**
-  The ID of the call set this variant call belongs to.
-  If this field is not present, the ordering of the call sets from a
-  `SearchCallSetsRequest` over this `VariantSet` is guaranteed to match
-  the ordering of the calls on this `Variant`.
-  The number of results will also be the same.
+  The ID of the call set this `Genotype` belongs to. Either this field
+  or callSetName must be set.
   */
   union { null, string } callSetId;
 
   /**
-  The name of the call set this variant call belongs to.
-  If this field is not present, the ordering of the call sets from a
-  `SearchCallSetsRequest` over this `VariantSet` is guaranteed to match
-  the ordering of the calls on this `Variant`.
-  The number of results will also be the same.
+  The name of the call set this `Genotype` belongs to. Either this
+  field or callSetId must be set.
   */
   union { null, string } callSetName = null;
 
   /**
-  The genotype of this variant call. Each value represents either the value
-  of the referenceBases field or is a 1-based index into alternateBases.
-  If a variant had a referenceBases field of "T", an alternateBases
-  value of ["A", "C"], and the genotype was [2, 1], that would mean the call
-  represented the heterozygous value "CA" for this variant. If the genotype
-  was instead [0, 1] the represented value would be "TA". Ordering of the
-  genotype values is important if the phaseset field is present.
+  What is the ID of this `Genotype`? Unique within the repository.
```

Comment:
It would be better to follow the existing pattern for documenting these attributes. i.e. `/** The ID of this Genotype. Unique within the repository. */`. I find it confusing to state the documentation for these attributes in the form of questions. Please rephrase these as declarative statements. (See above the documentation for `Segment.start`)